contact.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
In your last Pull Request, I had suggested keeping the Contact in the Footer instead of a separate page